src/App.js

```diff
@@ -17,8 +18,38 @@ const tabs = [
   },
 ];
 
-const App = () => (
-  <h1>React tabs</h1>
-);
+const preparedTabs = tabs.map((tab, index) => ({
+  ...tab,
+  id: index,
+}));
+
+class App extends React.Component {
+  state = {
+    visibleTabs: preparedTabs,
+    content: '',
+  }
+
+  onChange = (tabId) => {
+    this.setState(state => ({
+      content: state.visibleTabs.find(tab => tab.id === tabId).content,
+    }));
+  }
+
+  render() {
+    const { visibleTabs, content } = this.state;
+
+    return (
+      <div className="App">
+        <h1>React tabs</h1>
+        <Tabs
+          tabs={visibleTabs}
+          onChange={this.onChange}
+          content={content}
```

Comment:
You can pass selected tab id to the Tabs component instead of the tab content, as ids are unique, and content may be the same.